src/tree.rs

```diff
@@ -37,6 +37,12 @@ pub struct Tree<Pane> {
 
     /// All the tiles in the tree.
     pub tiles: Tiles<Pane>,
+
+    /// Maximum height
+    height: f32,
+
+    /// Maximum width
```

Comment:
These docstrings are inaccurate. These are the exact width/height, if finite.

Reply:
done